lib/ui/views/login_code/login_code_view.dart

```diff
@@ -35,12 +35,9 @@ class LoginCodeView extends StackedView<LoginCodeViewModel>
     LoginCodeViewModel viewModel,
     Widget? child,
   ) {
-    return PopScope(
```

Comment:
This one I'm not 100% sure, but on the other hand there's no harm on allowing user to navigate back from here. Maybe it should stay tho to avoid accidental navigations when trying to open email app. WDYS @JElgar I don't have strong opinion on leaving or removing it 😄

Reply:
Hmm yeah good question, I'll pull the app and have a play...